packages/angular_devkit/core/node/resolve.ts

```diff
@@ -50,17 +51,22 @@ function _getGlobalNodeModules() {
 
   if (process.env.PREFIX) {
     globalPrefix = process.env.PREFIX;
-  } else if (process.platform === 'win32') {
-    // c:\node\node.exe --> prefix=c:\node\
-    globalPrefix = path.dirname(process.execPath);
   } else {
-    // /usr/local/bin/node --> prefix=/usr/local
-    globalPrefix = path.dirname(path.dirname(process.execPath));
-
-    // destdir only is respected on Unix
-    const destdir = process.env.DESTDIR;
-    if (destdir) {
-      globalPrefix = path.join(destdir, globalPrefix);
+    const npmPrefix = _getNpmPrefix();
+    if (npmPrefix != null) {
+      globalPrefix = npmPrefix;
+    } else if (process.platform === 'win32') {
+      // c:\node\node.exe --> prefix=c:\node\
+      globalPrefix = path.dirname(process.execPath);
+    } else {
+      // /usr/local/bin/node --> prefix=/usr/local
+      globalPrefix = path.dirname(path.dirname(process.execPath));
```

Comment:
I think this might be why it's failing on circle. Their execPath might be different.